### DISCUSSION TITLE
about changing fonts

### DISCUSSION THREAD

**Discussion by nichroot** [end-4/dots-hyprland#867] [Q&A] [answered]
so i changed my font through gnome-tweaks and most of them changed, but there's still some part of ags that still uses the old fonts, mainly the clock part in the bar and some other widgets parts, can anyone help me change the font there too?

**Comment by end-4** [comment, marked as answer]
For the sake of interface readability I made most of the interface try to use one of a few known clean sans fonts, but I might be missing some spots so it isn't 100% consistent
To make sure the interface completely respects your default font, edit `/home/end/.config/ags/scss/_lib_mixins.scss`, specifically  the `@mixin mainfont` block. Change the `font-family` property to contain only `sans-serif`, which should be your system's sans font

**Comment by nichroot** [comment]
got it, thanks!
